README.md

```diff
@@ -105,6 +105,37 @@ func main() {
 }
 ```
 
+### Custom comparision function
+
+```go
+package main
+
+import(
+    "net/http"
+
+    "github.com/goji/httpauth"
+)
+
+func DummyBasicAuth() func(http.Handler) http.Handler {
+    opts := httpauth.AuthOptions{
+        Realm: "Restricted",
+        AuthFunc: func(u,p string) bool { return true },
```

Comment:
We should be providing useful examples here—just having a "dummy" one that returns true by default isn't helpful to beginners. We should show them how to create an actual `AuthFunc` here.